gadgets/trace_exec/test/trace_exec_test.go

```diff
@@ -98,13 +99,16 @@ func TestTraceExec(t *testing.T) {
 		commonDataOpts = append(commonDataOpts, utils.WithK8sNamespace(ns))
 	}
 
+	runnerOpts = append(runnerOpts, igrunner.WithFlags("--paths", "--verify-image=false"))
```

Comment:
The --verify-image=false is required for me, since otherwise tests fail with:
Error: fetching gadget information: initializing and preparing operators: instantiating operator "oci": ensuring image: verifying image "trace_exec:sefitufan-addingPaths": getting signing information: getting signature: getting signature bytes: ghcr.io/inspektor-gadget/gadget/trace_exec:sha256-9b2b5ff0a0387979d8ee3ebc18f69eaf5a2c60d2ecbdcd5a96e8c03096ced541.sig: not found

Should I leave this filter?

Reply:
For your own local tests you need that flag, since the gadget is not signed by the IG private key. In the PR CI we sign the image and also want to verify it.

So PR: no flag - local with flag

Reply:
It's also possible to use `IG_FLAGS="--verify-image=false"` when running the tests locally.